docs/en/query_language/functions/array_functions.md

```diff
@@ -789,7 +789,35 @@ SELECT arrayReverse([1, 2, 3])
 
 Synonym for ["arrayReverse"](#array_functions-arrayreverse)
 
-[Original article](https://clickhouse.yandex/docs/en/query_language/functions/array_functions/) <!--hide-->
+## flatten {#flatten}
+
+Converts array of arrays to a flat array.
```

Comment:
Also we need to explicitly specify that flattened array contains all elements from all source arrays.